### PR TITLE
DESENG-691 Update forest planning URL

### PR DESCRIPTION
**Pull request by Baelx**
Prettier updating the formatting of the changelog, FYI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "base-app",
-  "version": "1.11.0",
+  "version": "1.13.0",
```
Comment (NatSquared) [inline package.json:3]: any reason we're going up 2 versions?

File: src/app/process/process.component.html [hunk 2]
```diff
@@ -42,7 +42,7 @@ <h1 tabindex="0">Forest Landscape Planning</h1>
           </p>
           <p>
             For more information on forest landscape planning, please visit the B.C. government’s
-             <a href="https://www2.qa.gov.bc.ca/gov/content?id=A6A208C8A4224EA1BEEC38B337DDA1B8">forest landscape planning homepage.</a>
+             <a href="https://www2.gov.bc.ca/gov/content?id=A6A208C8A4224EA1BEEC38B337DDA1B8">forest landscape planning homepage.</a>
```
Comment (NatSquared) [inline src/app/process/process.component.html:45]: Corresponds with the permalink on https://www2.gov.bc.ca/gov/content/industry/forestry/managing-our-forest-resources/forest-landscape-plans 👍

**Review by NatSquared — APPROVED** [review]
Standard content update; looks good

File: package.json (hunk 1 above)
Comment (Baelx) [reply package.json:3]: Great question - I must have forgotten to update the version last time as it was out of sync with the Changelog. I brought it up to speed in this PR

File: src/app/process/process.component.html (hunk 2 above)
Comment (Baelx) [reply src/app/process/process.component.html:45]: Observant of you to notice!

**Pull request merged**